src/modules/navigator/navigator_params.c

```diff
@@ -121,11 +121,39 @@ PARAM_DEFINE_FLOAT(NAV_MC_ALT_RAD, 0.8f);
  * @value 1 Warn only
  * @value 2 Return mode
  * @value 3 Land mode
+ * @value 4 Position Hold mode
  *
  * @group Mission
  */
 PARAM_DEFINE_INT32(NAV_TRAFF_AVOID, 1);
 
+/**
+ * Set NAV TRAFFIC AVOID RADIUS MANNED
```

Comment:
@LowOrbitIonCannon Can we have a new PR to change this to more natural formatting?

> Set avoidance radius/distance for manned traffic (ADS-B/FLARM).